src/services/google.json

```diff
@@ -23,11 +23,11 @@
   "tosback2": {
     "definitions": {
       "name": "Key terms",
-      "url": "https://www.google.com/intl/en/policies/privacy/key-terms/"
```

Comment:
Argh. So next time, if I `pull` before doing the import it should be ok? Or it isn't something we should worry about?